showItemPageBE/src/main/java/com/talentpath/shamazin/showItemPage/models/Item.java

```diff
@@ -21,6 +21,9 @@
     @OneToMany(mappedBy="item", cascade=CascadeType.ALL)
     private List<ProductPhoto> productPhotos;
 
+    @OneToMany(mappedBy = "question", cascade=CascadeType.ALL)
```

Comment:
mappedBy="itemFamily" (that's what you named your variable in itemFamily where you did the ManyToOne relationship) -- mappedBy is what we use to tell Hibernate which variable we are using to represent parent class in our child class =)